hack/gen-crd.sh

```diff
@@ -19,3 +19,10 @@ install_crd \
   "config/crd/bases/metal3.io_provisionings.yaml" \
   "manifests/0000_30_cluster-baremetal-operator_00_metal3provisioning.crd.yaml"
 
+# install bmh crd
+tmpfile="$(mktemp bmhcrd-XXXXXXXX)"
+curl -o $tmpfile https://raw.githubusercontent.com/openshift/baremetal-operator/master/deploy/crds/metal3.io_baremetalhosts_crd.yaml
```

Comment:
I would move the above two lines into a separate script to be invoked manually

Reply:
Curious why we want to keep it separate.

Reply:
Well, this script is part of the build system, and as a general approach I'd prefer to ensure the reliability of the build, given that the internet access (or even the target repo itself) could not be granted to be always  available when building (for example, from your disconnected laptop). Decoupling it will make the relationship loosely coupled, so with fewer chances of affecting the build process. The CRD also is not updated very frequently, so a manual triggering of the script could be enough in this case.

Reply:
@andfasano why do you think BMH CRD should be updated manually?

Reply:
Just to clarify, I meant that the update script (to copy the BMH CRD from the BMO repo into the `/manifests` folder) should be decoupled from the the `hack/gen-crd.sh`. Not necessarily invoked manually, there could be ie a periodic job on the CI for invoking it.

Reply:
OK. Makes sense. Let me know what you guys think of the latest commit.